app/src/main/java/org/simple/clinic/teleconsultlog/teleconsultrecord/screen/TeleconsultRecordUi.kt

```diff
@@ -9,4 +9,6 @@ interface TeleconsultRecordUi {
   fun setPatientTookMedicines(patientTookMedicines: Answer)
   fun setPatientConsented(patientConsented: Answer)
   fun renderPatientDetails(patient: Patient)
+  fun showDoneProgress()
+  fun hideDoneProgress()
```

Comment:
Can we drop the `Done` part of the method names? I don't think they add any additional meaning and are just noise at this point.

Reply:
That was intentional, `showProgress` & `hideProgress` sounded generic for the screen. That's why I added where we will be showing the progress. Should I continue renaming it? (or perhaps I can rename it to `showDoneButtonProgress` & `hideDoneButtonProgress`

Reply:
The way I am looking at this is that it is up to the screen whether to show this progress state on the done button or not and the UI renderer itself doesn't really care where it is shown, just that it is shown and hidden in some way.

Reply:
We shouldn't have to change these names if we decide to say, show a progress modal instead of doing it as part of the button.

Reply:
Ah okay, makes sense. Renaming the methods